src/controllers/ReportController.js

```diff
@@ -37,6 +93,127 @@ class ReportController {
     }
   }
 
+  /**
+   * @static
+   * @param {Object} req
+   * @param {object} res
+   * @return {object} report
+   */
+  static async reportArtAction(req, res) {
+    try {
+      const { reportId } = req.params;
+      const {
+        reporterId, artId, defaulterId
+      } = req.query;
+      await Art.destroy({
+        where: { id: artId }
+      });
+
+      const defaulter = await User.findOne({
```

Comment:
since you are only using the email, you can specify the attributes to ensure that you are not fetching unnecessary data.